sygnal/sygnal.py

```diff
@@ -235,7 +237,6 @@ def check_section(section_name, known_keys, cfgpart=config):
     check_section(
         "access", {"file", "enabled", "x_forwarded_for"}, cfgpart=config["log"]
     )
-    check_section("db", {"dbfile"})
```

Comment:
does the updated section not merit any checks?

Reply:
I've added some checks below. It's a bit *ugh* because we are checking for different things depending on the database being used, but it's better than nothing.

Reply:
I've since added checks that `name` and `args` are present.